pennylane/templates/subroutines/uccsd.py

```diff
@@ -191,7 +191,11 @@ def __init__(self, weights, wires, s_wires=None, d_wires=None, init_state=None,
         if init_state.dtype != np.dtype("int"):
             raise ValueError(f"Elements of 'init_state' must be integers; got {init_state.dtype}")
 
-        self._hyperparameters = {"init_state": init_state, "s_wires": s_wires, "d_wires": d_wires}
+        self._hyperparameters = {
+            "init_state": tuple(init_state),
+            "s_wires": tuple(tuple(w) for w in s_wires),
+            "d_wires": tuple(tuple(tuple(w) for w in dw) for dw in d_wires),
+        }
```

Comment:
Make 'em hashable. They are `Sequence`, `Sequence[Sequence]` and `Sequence[Sequence[Sequence]]]`, respectively.